src/pages/components/DeleteAdmin/useDeleteAdmin.ts

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import { deleteAdmin } from '@/api/adminActions';
+import { useTranslation } from 'react-i18next';
+
+export const useDeleteAdmin = (
+  adminId: number,
+  refreshAdmins: () => Promise<void>
+) => {
+  const [isOpened, setIsOpened] = useState<boolean>(false);
+  const { t } = useTranslation();
+
+  const handleOpen = (): void => setIsOpened(true);
+  const handleClose = (): void => setIsOpened(false);
+
+  const deleteAdminRequest = async (): Promise<void> => {
+    try {
+      await deleteAdmin(adminId);
+      await refreshAdmins();
+      handleClose();
+    } catch (error) {
+      console.error('Failed to delete admin:', error);
```

Comment:
same, no console. it should be error that user will see

Reply:
okay